### ISSUE TITLE
Double "minutes" on "No motion duration"

### ISSUE THREAD

**Issue by Mostalk** [al-one/hass-xiaomi-miot#1017]
![Screenshot_7](https://user-images.githubusercontent.com/9198186/216524493-4ccc1cbc-bfa9-4a85-9119-b0798a40d8ed.png)
![Screenshot_8](https://user-images.githubusercontent.com/9198186/216524546-4ab2a8a9-8a5d-492e-ad14-2b7eb2fa3725.png)

**Comment by al-one** [comment]
Is [`yeelink.light.nl1`](https://home.miot-spec.com/s/yeelink.light.nl1) ?

**Comment by Mostalk** [comment]
@al-one, nope. Is [miaomiaoce.sensor_ht.t2](https://home.miot-spec.com/s/lumi.sensor_motion.aq2) connected to [lumi.gateway.mgl03](https://home.miot-spec.com/s/lumi.gateway.mgl03)
